package.json

```diff
@@ -60,17 +61,16 @@
     "node-sass": "^4.11.0"
   },
   "dependencies": {
-    "@types/reserved-words": "^0.1.0",
```

Comment:
Since all of the built distribution is JavaScript there should be no need for the type definitions when using this package.